packs/feats/champion-dedication.json

```diff
@@ -99,6 +99,10 @@
                 "key": "GrantItem",
                 "uuid": "Compendium.pf2e.classfeatures.Item.Cause"
             },
+            {
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.classfeatures.Item.Champion's Aura"
+            },
```

Comment:
```suggestion
```
The Deity item already grants Champion's Aura. We already implemented this change as part of the PFS thing